ui/views_test.py

```diff
@@ -143,14 +145,17 @@ def test_mosaic_view(logged_in_client, settings):  # pylint: disable=redefined-o
     assert response.context_data['uswitchPlayerURL'] == 'https://testing_odl.mit.edu'
 
 
-def test_upload_dropbox_videos_authentication(logged_in_apiclient):
+def test_upload_dropbox_videos_authentication(mock_moira_client, logged_in_apiclient):
     """
-    Tests that only authenticated users can call UploadVideosFromDropbox
+    Tests that only authenticated users with collection admin permissions can call UploadVideosFromDropbox
     """
     client, user = logged_in_apiclient
     client.logout()
```

Comment:
I know this isn't part of this PR, but why are we using a logged-in API client, then immediately logging out? Why not just use the not-logged-in `apiclient`?